Moodlet/app/src/androidTest/java/com/cmput3owo1/moodlet/fragments/LoginUserTest.java

```diff
@@ -27,6 +29,10 @@
 /**
  * This tests for proper user login and failed user login with wrong
  * credentials and missing fields
+ *
+ * Note: The auto-login function in the login fragment must be commented out for the login tests
```

Comment:
Let's just comment out the auto-login forever :(

Reply:
ok :(